Service/src/main/java/Validator/Validator.java

```diff
@@ -0,0 +1,49 @@
+package Validator;
+
+import TIcket.ticketTypes;
+import org.example.SetClassId;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Validator implements SetClassId {
+    private static int classId;
+
+    private static Set<String> validValues = new HashSet<>();
+    static{
```

Comment:
why do we need here static block?

Reply:
I used a static block so that it was executed only once, on the first call, and then the already initialized HashSet was used

Reply:
Why don't do it in constructor?